pharmaid/src/main/java/coms4156/bytecoders/pharmaid/PharmaidApplication.java

```diff
@@ -0,0 +1,13 @@
+package coms4156.bytecoders.pharmaid;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+@SpringBootApplication
+public class PharmaidApplication {
+
+	public static void main(String[] args) {
```

Comment:
I think the formatting is odd, but it's ok for now. Once the checkstyle is added, we can always autoformat, but it should be the next step